therapeutic/security/ir.model.access.csv

```diff
@@ -0,0 +1,12 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink,perm_responsibility
+base_group_access_nh_clinical_shift,admin:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_base,1,0,0,0,0
+admin_access_nh_clinical_shift,admin:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_admin,1,1,1,1,0
+dev_access_nh_clinical_shift,dev:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_dev,1,1,1,1,0
+shift_coordinator_access_nh_clinical_shift,manager:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_ward_manager,1,1,1,0,0
+senior_manager_access_nh_clinical_shift,senior_manager:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_senior_manager,1,0,0,0,0
+adt_access_nh_clinical_shift,manager:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_adt,1,0,0,0,0
+kiosk_access_nh_clinical_shift,kiosk:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_kiosk,1,0,0,0,0
+hca_access_nh_clinical_shift,hca:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_hca,1,1,1,0,0
+nurse_access_nh_clinical_shift,nurse:access_shift,model_nh_clinical_therapeutic_level,nh_clinical.group_nhc_nurse,1,1,1,0,0
```

Comment:
Not sure if I got this right, but why do the Nurse and HCA get rights to create and write?